File: python/client/libs/response_system/IResponseWorker.py
import threading
import uuid
from response_system import ResponseManager, Response


class IResponseWorker(threading.Thread):
    def __init__(self):
        threading.Thread.__init__(self)
        self.responseManager = ResponseManager.ResponseManager()

    def sendData(self, data: str, msgUUID: uuid.UUID = None, encrypt: bool = True) -> Response.Response:
        pass

    def sendCommand(self, command: str, args: tuple = None) -> Response.Response:
        pass

File: java/libs/[ITS] Response System/src/net/kio/its/responsesystem/IResponseWorker.java
package net.kio.its.responsesystem;

import java.util.UUID;

public interface IResponseWorker {

    ResponseManager getResponseManager();

    Response sendData(String data);

    Response sendData(String data, boolean encrypt);

    Response sendData(String data, String responseUUID);

    Response sendData(String data, String responseUUID, boolean encrypt);

    Response sendData(String data, UUID responseUUID, boolean encrypt);

    Response sendCommand(String command, String... args);
}
